lib/less-node/lessc-helper.js

```diff
@@ -48,9 +48,13 @@ var lessc_helper = {
         console.log('  -rp, --rootpath=URL          Sets rootpath for url rewriting in relative imports and urls');
         console.log('                               Works with or without the relative-urls option.');
         console.log('  -ru, --relative-urls         Re-writes relative urls to the base less file.');
-        console.log('  -sm=on|off                   Turns on or off strict math, where in strict mode, math.');
-        console.log('  --strict-math=on|off         Requires brackets. This option may default to on and then');
-        console.log('                               be removed in the future.');
+        console.log('');
+        console.log('  -m=, --math=');
+        console.log('     always                    Less will eagerly perform math operations always.');
+        console.log('     parens-division           Math performed except for division (/) operator');
+        console.log('     parens-all                Math only performed inside parentheses');
```

Comment:
As I mentioned in a separate comment, I think `parens` is probably sufficient, but I don't mind either way, really.

Reply:
I'll do `parens` with `strict` as an alias.